packages/dd-trace/src/appsec/iast/analyzers/weak-hash-analyzer.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const { storage } = require('../../../../../datadog-core')
+const { IAST_CONTEXT_KEY } = require('./../index')
+const Analyzer = require('./vulnerability-analyzer')
+
+const INSECURE_HASH_ALGORITHMS = [
```

Comment:
I see that we can create a mistake in the future easily, if we add the algorithm in the list uppercased.
What do you think if we do something like that? (maintaining the original names)
```suggestion
const INSECURE_HASH_ALGORITHMS = ['RSA-MD5','RSA-SHA1', ...].map(x => x.toLowerCase())
```